### DISCUSSION TITLE
breaking changes in version 0.900.21? ("setInstanceCreatedCallback is not a function")

### DISCUSSION THREAD

**Discussion by acoyfellow** [firebase/firebase-js-sdk#4762] [Q&A] [answered]
I just tried upgrading from `0.900.19` to `0.900.21`, and i'm seeing:

```
Uncaught (in promise) TypeError: (intermediate value).setInstantiationMode(...).setInstanceCreatedCallback is not a function
    at registerAuth (register.ts:102)
    at index.ts:134
```

Simply rolling back to `0.900.20` side steps the issue. I'm still investigating, I'll see if i can get a minimal repo to show.

**Comment by Feiyang1** [comment, marked as answer]
Thanks for the report.
0.900.21 is broken right now. Please use 0.900.20 instead. We will make another release later today.

**Reply by Feiyang1** [reply]
This is fixed in 0.900.23.
